### DISCUSSION TITLE
Is it possible to have two rows in the QEditor toolbar?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#10816] [General - Components / Directives / etc]
I am trying to have two lines in the QEditor toolbar but the [documentation ](https://quasar.dev/vue-components/editor)does not mention the possibility.

Is this something that is doable?

**Comment by ontwikkelfabriek** [comment]
@wsw70 

> I am trying to have two lines in the QEditor toolbar

What do you mean with this exactly?

**Reply by wsw70** [reply]
It is to have a toolbar with two rows, one above this other. As far as I can tell, the docs do not mention it but maybe I missed that part. 

Artistically speaking, this is something like this:

![image](https://user-images.githubusercontent.com/1345886/134774080-d2b89f3f-4c7d-4040-aa09-6cd7f41f1166.png)

The upper row has 6 buttons, the lower one  - 3

**Reply by ontwikkelfabriek** [reply]
@wsw70 

I guess not, no `slots` to do anything.
